contents/docs/user-guides/snippets/groups-ingestion-segment.mdx

```diff
@@ -0,0 +1,28 @@
+Not all libraries support group analytics yet, but you can work around this issue by sending events in specific formats.
+
+This section uses our [Capture APIs](https://posthog.com/docs/api/post-only-endpoints) for examples but you can adapt this
+approach to any library.
+
+#### Capturing events with groups
+
+```js
+analytics.track('[event name]', {
+    "$groups": {
+        "company": "id:5"
+    }
+})
+
+```
+
+#### Updating group properties
+
+```js
+analytics.track('$groupidentify', {
```

Comment:
Have you tested this - will this work? Our APIs expect a distinct_id, is that provided somehow differently in segment?

Reply:
Yeah this example is for their js/mobile libraries. Have clarified by adding examples of backend calls.